charts/postgres-db-sync/templates/sync-job.yaml

```diff
@@ -38,11 +38,16 @@ spec:
                 export OUTPUT_DB_PASS=${PGDBSYNC_PASSWORD_{{ .outputDB }}};
                 export OUTPUT_SCHEMA='{{ .outputSchema }}';
                 export OUTPUT_TABLE='{{ .outputTable }}';
+                export OUTPUT_LIST_COLUMN_NAMES_DATA_TYPES='{{ .outputListColumnNamesDataTypes }}';
                 export TIMESTAMP_COLUMN='{{ .timestampColumn }}';
                 export TRUNCATE_TARGET_TABLE='{{ $.Values.cronjob.truncateTargetTable }}';
                 export LAG_MINUTES='{{ $.Values.cronjob.lagMinutes }}';
 
-                /usr/local/bin/table2table_sync.sh
+                {{ if eq $.Values.syncMethod "tempTables" }}
+                /usr/local/bin/temp_table_sync.sh
+                {{ else }}
```

Comment:
can you also do a check for the other string, and raise an error if the value is unknown?

Reply:
I did this now but I could only do this within bash